<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseEditorDialog</class>
 <widget class="QDialog" name="DatabaseEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ql_constituentName">
       <property name="text">
        <string>Constituent name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qle_constituentName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DatabaseTableView" name="qtbv_constituents"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="qpb_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_edit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="qpb_allCompounds">
       <property name="text">
        <string>All compounds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatabaseTableView</class>
   <extends>QTableView</extends>
   <header>src/ui/databasetableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
